examples/dmabuf-capture.c

```diff
@@ -24,6 +26,15 @@ struct wayland_output {
 	AVRational framerate;
 };
 
+struct fifo_buffer {
+	AVFrame **queued_frames;
+	int num_queued_frames;
+	int max_queued_frames;
+	pthread_mutex_t lock;
+	pthread_cond_t cond;
+	pthread_mutex_t cond_lock;
+} fifo_buffer;
```

Comment:
This creates a struct type named fifo_buffer and also a static instance of that struct called fifo_buffer